numpyro/distributions/discrete.py

```diff
@@ -618,11 +632,31 @@ def infer_shapes(logits, total_count):
         return batch_shape, event_shape
 
 
-def Multinomial(total_count=1, probs=None, logits=None, *, validate_args=None):
+def Multinomial(
+    total_count=1, probs=None, logits=None, *, total_count_max=None, validate_args=None
+):
+    """Multinomial distribution.
+
+    :param total_count: number of trials. If this is a JAX array,
+        `total_count_max` is required to specify.
```

Comment:
=> it is required to specify `total_count_max`.

Reply:
done